Sources/SwiftParser/Declarations.swift

```diff
@@ -1884,14 +1838,15 @@ extension Parser {
 
         // Next, parse effects specifiers. While it's only valid to have them
         // on 'get' accessors, we also emit diagnostics if they show up on others.
-        let asyncKeyword: RawTokenSyntax?
-        let throwsKeyword: RawTokenSyntax?
-        if self.at(anyIn: EffectsSpecifier.self) != nil {
-          asyncKeyword = self.parseEffectsSpecifier()
-          throwsKeyword = self.parseEffectsSpecifier()
+        var asyncOrReasync: RawTokenSyntax? = nil
+        var throwsOrRethrows: RawTokenSyntax? = nil
```

Comment:
I think the general paradigm in the rest of the compiler is to leave these uninitialized here 

```suggestion
        var asyncOrReasync: RawTokenSyntax?
        var throwsOrRethrows: RawTokenSyntax?
```

Reply:
Think I had something different here originally, but I can change these to `let` 👍